.storybook/themes/news.scss

```diff
@@ -835,6 +875,9 @@
 					padding: 0 var(--global-spacing-6) 0 0,
 					display: flex,
 				),
+				share-bar: (
+					display: flex,
+				),
```

Comment:
```suggestion
```

Reply:
@JackHowa : As said above, this is to set display the `<Stack/>` container from `none` on `default()` to `flex` on `desktop()`.